generator.rb

```diff
@@ -0,0 +1,85 @@
+def is_vowel?(letter)
+    letter.each_char do |char|
```

Comment:
`letter` is going to represent one letter. Why do you need to do `each_char` and loop through it if it's one letter?

Reply:
Changed the parameter name from 'letter' to 'word' since method takes in a word and checks if each character is a vowel or not.